docs/examples/plot_linear_quickstart.py

```diff
@@ -13,9 +13,9 @@
 ######################################################################
 # To start, we need to read and preprocess the input data:
 
-preprocessor = linear.Preprocessor(data_format="txt")
-
-datasets = preprocessor.load_data("data/rcv1/train.txt", "data/rcv1/test.txt")
+datasets = linear.load_dataset("txt", "data/rcv1/train.txt", "data/rcv1/test.txt")
+preprocessor = linear.Preprocessor()
+datasets = preprocessor.fit_transform(datasets)
```

Comment:
users may not be familiar with the "fit" and "transform" names used by sklearn. May need more explanation